packages/document/module-doc/docs/en/api/config/build-config.mdx

```diff
@@ -166,6 +159,21 @@ Set unmatched svg files
 - **Type**: `string | RegExp | (string | RegExp)[]`
 - **Default**: `undefined`
 
+## asset.svgr.exportType
+
+use named exoprt in case
+
+- **Type**: `'named' | 'default'`
+- **Default**: `default`
+
+when it is 'named', use the following syntax:
+
+```js title="index.ts"
+import { ReactComponent } from './logo.svg';
+```
+
+The named export defaults to `ReactComponent`, and can be customized with the `asset.svgr.namedExport`.
```

Comment:
Where can we find the document of `asset.svgr.namedExport`

Reply:
I have not found the document in svgr document, it only mentioned in https://react-svgr.com/docs/webpack/#use-with-url-loader-or-file-loader

Reply:
how about add it to module-tools document